src/_tests_/App.test.js

```diff
@@ -0,0 +1,3 @@
+it("Has no tests", () => {
+  expect(true).toBe(true);
```

Comment:
Do we even need this file honestly? Maybe we should just delete it 🤷‍♀

Reply:
@constancecchen If there are now no tests, jest will fail. So CI was failing for this project. Since we may add real tests in the future, I thought it might be easier to keep CI in place and simply stub out a fake test for now.

Reply:
Haha whoa, that's wild. Thanks for the context!